core/unbound_method.rbs

```diff
@@ -78,7 +100,7 @@ class UnboundMethod
   #     m.call # => "bar"
   #     n = m.clone.call # => "bar"
   #
-  def clone: () -> self
+  def clone: () -> instance
```

Comment:
`self`?

Reply:
I was thinking `self` means "the exact same object" (i.e. with `def foo: () -> self`, `self.foo.equal?(self)` wouldalways be true), whereas `instance` means an instance of `Binding`.